core-common/src/main/java/org/zalando/nakadi/domain/EventPublishingStep.java

```diff
@@ -5,5 +5,6 @@
     VALIDATING,
     ENRICHING,
     PARTITIONING,
+    AUTHORIZING,
```

Comment:
unfortunately, `EvnetPublishingStep` is Enum, changing will introduce breaking change
https://nakadi.io/manual.html#definition_BatchItemResponse*step

Reply:
:((

Reply:
Thanks for pointing out. :+1: , We have a few options I believe:

1. a work-around would be to use an existing step (`VALIDATING` or `NONE`). Given the `event_owner_header` definition is part of event type definition (like `schema`), and it is checked against the actual event (like event against `schema`), I believe `VALIDATING` step could be a work around. (with update in documentation). What do you think/suggest? 

2. introduce breaking change: This might not be an option, but if existing clients don't use the step field, we could probably introduce this. I'll check this for major clients.

Reply:
first option using `VALIDATING` step sounds good to me
the field `step` looks overwhelming, whatever it is, user has to go and read the detailed message, which Nakai provides. Processing based on that field can not be applied in automated way, it is done based on response codes from Nakadi.

Reply:
Fixed this in https://github.com/zalando/nakadi/pull/1147/commits/dfb2c00f98ce6224e624ee6c28445612a6c82ccb